ftw/contacts/contact.py

```diff
@@ -0,0 +1,237 @@
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    dexteritytextindexer.searchable('postal_code')
+    postal_code = schema.TextLine(
+        title=_(u'label_postal_code', default=u'Postal code'),
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    dexteritytextindexer.searchable('city')
+    city = schema.TextLine(
+        title=_(u'label_city', default=u'City'),
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    show_placemark = schema.Bool(
+        title=_(u'label_showplacemark', default=u'Show on map'),
+        required=False)
+
+    form.mode(country='hidden')
+    country = schema.TextLine(
+        title=_(u'label_country', default=u'Country'),
+        required=False,
+        default=u"Schweiz")
+
+    dexteritytextindexer.searchable('email')
+    email = RFC822MailAddress(
+        title=_(u'label_email', default=u'E-Mail'),
+        missing_value=u"",
+        required=False)
+
+    dexteritytextindexer.searchable('phone_office')
+    phone_office = schema.TextLine(
+        title=_(u'label_phone_office', default=u'Office phone number'),
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    dexteritytextindexer.searchable('phone_mobile')
+    phone_mobile = schema.TextLine(
+        title=_(u'label_phone_mobile', default=u'Mobile phone number'),
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    dexteritytextindexer.searchable('fax')
+    fax = schema.TextLine(
+        title=_(u'label_fax', default=u'Fax number'),
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    www = schema.URI(
+        title=_(u'label_www', default=u'www'),
+        missing_value=u"",
+        required=False)
+
+    image = NamedBlobImage(
+        title=_(u'label_image', default=u'Image'),
+        missing_value=u"",
+        required=False)
+
+    academic_title = schema.TextLine(
+        title=_(u'label_academic_title', default=u'Academic title'),
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    dexteritytextindexer.searchable('function')
+    function = schema.TextLine(
+        title=_(u'label_function', default=u'Function'),
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    dexteritytextindexer.searchable('department')
+    department = schema.TextLine(
+        title=_(u'label_department', default=u'Department'),
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    dexteritytextindexer.searchable('salutation')
+    salutation = schema.TextLine(
+        title=_(u'label_salutation', default=u'Salutation'),
+        missing_value=u"",
+        default=u"",
+        required=False)
+
+    dexteritytextindexer.searchable('text')
+    text = RichText(
+        title=_(u'label_text', default=u'Text'),
+        missing_value=u"",
+        default=u"",
```

Comment:
The RichText field does not contain strings, thus `missing_value` and `default` are invalid, causing transaction problems in the test.
Please remove `missing_value` and `default` and let the field do the job, it will fix the test isolation problems.